client/client-multi/client-c/tools/kaa_encryption/rsa_key_gen/kaa_rsa_key_gen.h

```diff
@@ -74,6 +74,11 @@ typedef struct {
 
 int rsa_genkey(mbedtls_pk_context *pk);
 
+/* Use this function to extract RSA keys from mbedtls_pk_context.
```

Comment:
Covert this comment to the doxygen-like comment and add doxygen comments everywhere in this file